### ISSUE TITLE
README Update

### ISSUE THREAD

**Issue by nsryan2** [arfc/phlox#3]
It looks like you've got a lot of really cool calculations and modeling ideas going on here, but it's a little hard to parse what things are for. I think your repo could benefit from some additional notes in the following places:

*These suggestions are all based off of what is currently on the [fix branch](https://github.com/ZoeRichter/phlox/tree/fix) in @ZoeRichter's fork, which (from what I can tell) is ahead of the main branch in this repo.*

`phlox/`: 
1. Could use information about implementation and requisite software for replication. You mention Serpent 2 in a sub-folder, maybe tossing in our group's [guide for getting a serpent license](http://arfc.github.io/manual/guides/getting_serpent_license) would be useful (though I'm not sure how useful that is for someone not in the group).
2. You track `.ipynb_checkpoints` in a couple different places in this directory, these are not typically tracked to reduce clutter.
3. There are a couple parts of the repo that use -20 and -200 to differentiate models, it would be useful to describe those differences at the top level so people know how to interpret it when they encounter it later on. 

`phlox/reactors/`: 
1. I could easily be mis-understanding the way you structured this, but it seems like you’re combining the design and analysis in the same reactors folder. I’m not sure if you’ve started the analysis (other than some of the plots you have under `full-core/post-proc`), but it might be nice to have that exist separately from  `phlox/reactors/`.
2. It would also be nice if this README summarized the three different prongs (fuel-fidelity, full-core, and single-pebble) a little bit to aid in navigation.

`phlox/reactors/fuel-fidelity/`:
1. I’m not the most familiar with serpent, but I just wanted to check to see if there’s any other information someone would need to run these simulations (like a specific library or something). If those exist somewhere else in the repo, or are a public resource, it might be useful to specify what they are.

`phlox/reactors/full-core/`:
1. This part of the directory could benefit from a README.
2. Something I noticed is that the htgr-20 and htgr-200 folders are very different in their contents. This could easily be that you’re still working on the htgr-200 one, but some of the things (like the tests or slice information) might be useful to have about both.

`phlox/reactors/full-core/htgr-20/`:
1. This folder could really benefit from a README detailing things like:
    A. What the 7 digit number folders are
    B. What the `slice-#-#` is (it’s an intuitive way to name them, I’m 95% sure it means the slice of the core from say 0 to 60 degrees right?)
2. It would be a pain, but your repo might benefit from having a folder for tests, one for the 7 digit number folders, the slices, and some of the other categories in this folder.

`phlox/reactors/full-core/pbbls/`:
1. A README could help clarify this, but are the `pbbls#.inp` files generic and the `pebbles-20-#` for the `-20` model?
2. Are these pebble input files only useful for the full-core, could they be used for more generic stuff as well? If so, it would be useful to have them higher up in the directory.

`phlox/reactors/single-pebble/`:
1. This folder could benefit from a README detailing what’s going on, and the goals for each of the `burn-#` folders.

You've got a lot of cool results, and I can't wait to see the final product.

**Comment by ZoeRichter** [comment]
I think you have a good point about the READMEs!  Thanks!

**Comment by nsryan2** [comment]
You're very welcome! If there's anything I can do, even though I'm not super familiar with the project, please let me know.
